src/sampler.jl

```diff
@@ -1,7 +1,8 @@
+using LogarithmicNumbers
 # Valid balancing functions
 
 metropolis_balance(t) = min(one(t), t)
-sqrt_balance(t) = sqrt(t)
+sqrt_balance(t) = t^(0.5)
```

Comment:
```suggestion
sqrt_balance(t) = sqrt(t)
```

Fixed by https://github.com/cjdoris/LogarithmicNumbers.jl/pull/19#issuecomment-1925811277

Reply:
If we leave sqrt then we can merge but we shouldn’t th a new version until upstream is released

Reply:
ok